.circleci/config.yml

```diff
@@ -112,6 +112,7 @@ jobs:
             - packages/*/dist
             - packages/*/build
             - packages/nitro-protocol/artifacts
+            - packages/nitro-protocol/typechain
```

Comment:
Clarifying circle config:

This is writing these directories into the filesystem of circleCI's vm, and not persisting in the 'artifact' sense? It writes these to make them available for the later test job.

Reply:
Exactly!